managed_resource/plugin/collector.yaml

```diff
@@ -0,0 +1,11 @@
+---
+capability: {}
+image: YOUR_DOCKERHUB_REPO/plugin-nhncloud-inven-collector
+labels: []
+name: NHN Cloud Collector
+plugin_id: plugin-nhncloud-inven-collector
+provider: nhncloud
+registry_type: DOCKER_HUB
+resource_type: inventory.Collector
+tags:
+  icon: https://rlhezzw2e.toastcdn.net/images/contents/nhncloud_symbol.png
```

Comment:
For icon, use https://github.com/cloudforet-io/static-assets/tree/master/providers/nhncloud